backend/open_webui/main.py

```
@@ -115,6 +115,8 @@
     BYPASS_MODEL_ACCESS_CONTROL,
     RESET_CONFIG_ON_START,
     OFFLINE_MODE,
+    WEBUI_BASE_PATH,
+    WEBUI_URL,
```

Ruff (0.8.2)

Redefinition of unused `WEBUI_URL` from line 114

Remove definition: `WEBUI_URL`

(F811)